MR4/expression/call.3.mr

```diff
@@ -0,0 +1,99 @@
+## MR4 compiler - Call expression
+
+## Function call expression
+class Call-expression(Expression)
```

Comment:
I think we talked about this, but I suggest classes have a CamelCase convention.